ci/properties/matlab.properties.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "MATLAB",
+    "description": "Build and test with MATLAB",
+    "iconName": "matlab",
+    "creator": "MathWorks",
```

Comment:
I don't see any of the other properties files with this key. Where did it come from?

Reply:
It is listed on the starter-workflows [README](https://github.com/actions/starter-workflows#valid-properties) page itself. It is present in some other templates too outside the ci directory, [example](https://github.com/actions/starter-workflows/blob/02af783829f3f5b38e36ff3145c01c38ca02ede3/deployments/properties/azure-functions-app-nodejs.properties.json#L4).